homeassistant/components/onkyo/media_player.py

```diff
@@ -309,19 +309,15 @@ def process_update(self, update):
                 self._attributes.pop(ATTR_VIDEO_INFORMATION, None)
                 self._attributes.pop(ATTR_PRESET, None)
                 self._attributes.pop(ATTR_VIDEO_OUT, None)
+
```

Comment:
I don't think we need the extra space here

Reply:
As far as I know this is the black formatting we must comply to?
(It checks the formatting when I do a commit)

Reply:
Got it